Application Settings/StudentRegestrationPage.page

```diff
@@ -0,0 +1,32 @@
+<apex:page standardController="Student__c" extensions="StudentRegestrationController"
+    language="{!languageOption}">
+    <apex:form>        
+    <apex:pageBlock id="studentPage">
+            <apex:selectList value="{!languageOption}" size="1"
+                multiselect="false">
+                <br/><br/>
+                <apex:pageMessages id="message">
+                </apex:pageMessages>
+                <apex:selectOptions value="{!language}" />
+                    <apex:actionsupport event="onchange" rerender="studentPage" />
+                </apex:selectList>
```

Comment:
Indent one tab back for this line as well as below lines. Please match with the starting tags